### ISSUE TITLE
Raise the music volume range

### ISSUE THREAD

**Issue by GrimPixel** [drwhut/tabletop-club#302]
**Is your feature request related to a problem? Please describe.**
When the audio file is played inside the game it is much quieter than the file played outside the game.

**Describe the solution you'd like**
Set the volume upper limit higher.

**Comment by drwhut** [comment]
Yeah, this was a deliberate decision I made to set the upper limit of the game's audio to -6 dB (about half the amplitude of what the system considers "maximum volume"), since I realized that someone could potentially import an audio track at max volume, and when it is played in game it could potentially blow out someone's ear drums if their volume settings were set to the maximum.

I'll label this issue as a question though, since it may be worth removing the upper limit as long as there's some form of safety net to replace it.

**Comment by GrimPixel** [comment]
Maybe an audio analyser? So it guarantees that all tracks have similar loudness.
